### ISSUE TITLE
Add Facebook Like Button

### ISSUE THREAD

**Issue by techieshark** [codeforamerica/MuniciPal#122]
This is similar to #94, and between this and 94 it may address the spirit of issue #120.

**Comment by michaelhwan** [comment]
@techieshark I don't think this is needed. Adding more social media features is nice but it can clutter things up on the webpage.

**Comment by techieshark** [comment]
I'll agree that I see this as less important than a share button.
